projects/packages/plans/src/class-current-plan.php

```diff
@@ -390,4 +398,39 @@ function_exists( 'wpcom_feature_exists' ) &&
 
 		return false;
 	}
+
+	/**
+	 * Retrieve site-specific features for Simple sites.
+	 *
+	 * See Jetpack_Gutenberg::get_site_specific_features()
+	 *
+	 * @return array
+	 */
+	public static function get_simple_site_specific_features() {
```

Comment:
@Automattic/jetpack-crew @Automattic/jetpack-garage this is where I need your feedback. I want to know whether this is OK to do here. 

[`Plans::get_plans()`](https://github.com/Automattic/jetpack/blob/c82adbddcf7f44d71e7e1a7e39a56ad00a193838/projects/packages/plans/src/class-plans.php#L27-L34) already does something similar.

Reply:
It looks OK to me, since we're making sure first that this is a Simple site environment. And I think copying the method is a valid approach to make sure we're leaving old methods in place.

Reply:
A followup might change the copied-from method to call through to this one instead of continuing to duplicate the logic, and maybe add an `@deprecated` to it.